### PR TITLE
OUT-733 Default order of tasks should be most recent at the top

### PR DESCRIPTION
**Pull request by aatbip**
### Changes

- [x] Adds `sortTaskByDescendingOrder` to `setTasks` in taskBoardSlice. 

### Testing Criteria

- [LOOM]()

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,7 +81,6 @@
   },
   "lint-staged": {
     "src/**/*.{ts,tsx}": [
-      "yarn lint:fix",
```
Comment (sazanrjb) [inline package.json:84]: what is the reason for remove this?

File: src/app/ui/TaskBoard.tsx [hunk 2]
```diff
@@ -21,9 +20,11 @@ import { Header } from '@/components/layouts/Header'
 import { FilterBar } from '@/components/layouts/FilterBar'
 import { CreateViewSettingsDTO } from '@/types/dto/viewSettings.dto'
 import { CustomLink } from '@/hoc/CustomLink'
+import { sortTaskByDescendingOrder } from '@/utils/sortTask'
 
 export const TaskBoard = () => {
   const { workflowStates, tasks, token, filteredTasks, view, filterOptions } = useSelector(selectTaskBoard)
+  console.log('tasks', tasks)
```
Comment (saugatmaharjan) [inline src/app/ui/TaskBoard.tsx:27]: console.log

File: package.json (hunk 1 above)
Comment (aatbip) [reply package.json]: it was removed by mistake. added in the commit below.

File: src/app/ui/TaskBoard.tsx (hunk 2 above)
Comment (aatbip) [reply src/app/ui/TaskBoard.tsx]: removed in the commit below

**Review by sazanrjb — APPROVED** [review]

**Pull request merged**